documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,218 @@
+In situations such as these, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
+The `KafkaTopic` resources must be recovered before a cluster is started so that the topics are not deleted by the Topic Operator.
+
+If you do not use the the Topic Operator, you just need to recover the `PersistentVolumeClaim` (PVC) resources.
+
+.Before you begin
+
+In this procedure, it is essential that PVs are mounted into the correct PVC to avoid data corruption.
+A `volumeName` is specified for the PVC and this must match the name of the PV.
+
+For more information, see:
+
+* xref:ref-persistent-storage-{context}#pvc-naming[Persistent Volume Claim naming]
+* xref:ref-jbod-storage-{context}#jbod-pvc[JBOD and Persistent Volume Claims]
+
+NOTE: The procedure does not include recovery of `KafkaUser` resources, which must be recreated manually.
+
+.Procedure
+
+. Check information on the PVs in the cluster:
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl get pv
+----
++
+Information is presented for PVs with data.
++
+Example output showing columns important to this procedure:
++
+[source,shell,subs="+quotes,attributes"]
+----
+NAME                                         RECLAIM POLICY
+pvc-5e9c5c7f-3317-11ea-a650-06e1eadd9a4c ... Retain
+pvc-5e9cc72d-3317-11ea-97b0-0aef8816c7ea ... Retain
+pvc-5ead43d1-3317-11ea-97b0-0aef8816c7ea ... Retain
+pvc-7e1f67f9-3317-11ea-a650-06e1eadd9a4c ... Retain
+pvc-7e21042e-3317-11ea-9786-02deaf9aa87e ... Retain
+pvc-7e226978-3317-11ea-97b0-0aef8816c7ea ... Retain
+
+CLAIM                                     STORAGECLASS
+myproject/data-my-cluster-zookeeper-1 ... gp2-retain
+myproject/data-my-cluster-zookeeper-0 ... gp2-retain
+myproject/data-my-cluster-zookeeper-2 ... gp2-retain
+myproject/data-0-my-cluster-kafka-0   ... gp2-retain
+myproject/data-0-my-cluster-kafka-1   ... gp2-retain
+myproject/data-0-my-cluster-kafka-2   ... gp2-retain
+----
++
+* _NAME_ shows the name of each PV.
+* _RECLAIM POLICY_ shows that PVs are _retained_.
+* _CLAIM_ shows the link to the original PVCs.
+* _STORAGECLASS_ shows the name of the storage class used for dynamic volume allocation.
+
+. Recreate the original namespace:
++
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl create namespace _myproject_
+----
+
+. Recreate the original PVC resource specifications, linking the PVCs to the appropriate PV:
++
+For example:
++
+[source,shell,subs="+quotes,attributes"]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: data-0-my-cluster-kafka-0
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Gi
+  storageClassName: gp2-retain
+  volumeMode: Filesystem
+  volumeName: *pvc-7e1f67f9-3317-11ea-a650-06e1eadd9a4c*
+----
+
+. Edit the PV specifications to delete the `claimRef` properties that bound the original PVC.
++
+For example:
++
+[source,shell,subs="+quotes,attributes"]
```

Comment:
@scholzj - do we want to show the whole example?

Reply:
TBH, I don't know. What do you think would make it eaiser to understand for you?

Reply:
As it's the only example of _kind: PersistentVolume_ we have in the doc, I will leave as is.